src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -400,15 +400,14 @@ export const veteranInformationViewField = (formData) => {
  *
  * @param {Disability} disability
  */
-export const disabilityOption = ({ diagnosticCode, name, ratingPercentage }) => {
+export const disabilityOption = ({ diagnosticCode, ratingPercentage }) => {
   // May need to throw an error to Sentry if any of these doesn't exist
   // A valid rated disability *can* have a rating percentage of 0%
   const showRatingPercentage = Number.isInteger(ratingPercentage);
 
   return (
     <div>
       {diagnosticCode && <h4>{getDiagnosticCodeName(diagnosticCode)}</h4>}
-      {name && <p className="diagnostic-text">{getDiagnosticText(name)}</p>}
```

Comment:
Turns out we don't actually use this; it's just what EVSS says we could display, but we wanted our own mapping so we could edit.